libnavui-base/src/main/java/com/mapbox/navigation/ui/base/lifecycle/UIComponent.kt

```diff
@@ -40,7 +40,7 @@ open class UIComponent : MapboxNavigationObserver {
         coroutineScope.cancel()
     }
 
-    protected inline fun <T> Flow<T>.observe(crossinline action: suspend (value: T) -> Unit) {
```

Comment:
This `observe` version is not available anymore. I initially thought that tests might be failing due to this change but checked the legacy code and shouldn't as they're using a `FlowCollector<T>` as we do now 🤔